### PR TITLE
Chore/update discoverydropdowntag

### PR DESCRIPTION
**Pull request by divyaocc**
Added code for tooltip info button  right next to each study filter category with additional data in DiscoveryDropdownTagViewer.tsx file under uc-cdis/data-portal/src/Discovery.

### CODE REVIEW

File: src/Discovery/DiscoveryDropdownTagViewer.tsx [hunk 1]
```diff
@@ -122,7 +124,34 @@ const DiscoveryDropdownTagViewer: React.FunctionComponent<DiscoveryTagViewerProp
...
               xxl={12}
             >
-              { tags }
+              <table >
+                <tr>
+                  <td >
+                  <Col >
+                  <Tooltip
+                  placement='left'
+                  overlay={category.tooltip}
```
Comment (paulineribeyre) [inline src/Discovery/DiscoveryDropdownTagViewer.tsx:133]: the `tooltip` field should be added to the docs at `docs/portal_config.md`

**Review by mfshao — COMMENTED** [review]
should also add similar feature into https://github.com/uc-cdis/data-portal/blob/master/src/Discovery/DiscoveryTagViewer.tsx because they are basically the same thing with different appearences

File: src/Discovery/DiscoveryDropdownTagViewer.tsx (hunk 1 above)
Comment (divyaocc) [reply src/Discovery/DiscoveryDropdownTagViewer.tsx:133]: I added  tooltip": "additional data", // optional string to add data when hovered on the info button at  portal_config.md

**Comment by ocshawn** [comment]
closing to cleanup prs if this is still an issue please address above cements and reopen

**Pull request closed without merging**